src/Converter_TEST.cc

```diff
@@ -2141,6 +2141,364 @@ TEST(Converter, Pose_16_to_17)
   EXPECT_STREQ("parent", jointLinkPoseElem->Attribute("relative_to"));
 }
 
+const std::string CONVERT_DOC_17_18 =
+  sdf::filesystem::append(PROJECT_SOURCE_PATH, "sdf", "1.8", "1_7.convert");
+
+/////////////////////////////////////////////////
+/// Test conversion unflattened world in 1.7 to 1.8
+TEST(Converter, World_17_to_18)
+{
+  // for ElementToString
+  using namespace sdf::SDF_VERSION_NAMESPACE;
```

Comment:
nit: I think this can just be `using namespace sdf`;

Reply:
3e0cd5d70fce3d7dbe32746c2f3ce526ab059baa